cityofaustin/atd-moped
# Show visual feedback when user needs to zoom to see selectable features

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10825

This PR adds an alert to show users when they need to zoom in to see the selectable CTN layers during the map step of creating or editing a component. The alert should only show when:
1. You are zoomed out beyond the level where we fetch data and show the CTN layers
2. You creating or editing a component and adding features with the map
3. You are not actively drawing component features

## Testing
**URL to test:** 

local or https://md-10825-zoom-alert--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Create a new project and go to the **Map** tab
4. Do not change the zoom of the map
5. Create a new component, fill out the form, and click **Continue**
6. When the map shows, notice that:
   - an alert shows in the top map bar telling you to zoom in to see selectable features
   - as you zoom in, you will reach a point where the CTN dashed lines or points layer appears and the alert disappears
7. Zoom back out to make the alert come back
8. Click on the draw tools in the top right to draw a line or point
9. When drawing, the alert should not show
10. You can try out the same steps with editing a component

---
#### Ship list
